pkg/cluster/config_test.go

```diff
@@ -188,3 +188,52 @@ func TestGetVXLANPort(t *testing.T) {
 		})
 	}
 }
+
+// TestGetDNS tests the DNS server IP generation from a given network CIDR
+func TestGetDNS(t *testing.T) {
+	type args struct {
+		ipRange string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    string
+		wantErr bool
+	}{
+		{
+			name:    "empty IP range",
+			args:    args{ipRange: ""},
+			want:    "",
+			wantErr: true,
+		},
+		{
+			name:    "invalid IP range",
+			args:    args{ipRange: "invalid"},
+			want:    "",
+			wantErr: true,
+		},
+		{
+			name:    "no IP in range",
+			args:    args{ipRange: "172.30.0.0/32"},
+			want:    "",
+			wantErr: true,
+		},
+		{
+			name:    "valid IP range",
+			args:    args{ipRange: "172.30.0.0/16"},
+			want:    "172.30.0.10",
+			wantErr: false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			// act
```

Comment:
I see a pattern of adding such comments all over place. I don't see any value in this other than stating the obvious and adding unnecessary verbosity. Comments should explain something that is not easily understood. Please remove all instances like `arrange`, `act`, `assert`.

Reply:
Addressed.